tools/sync_test_server/Dockerfile

```diff
@@ -2,21 +2,21 @@ FROM ubuntu:16.04
 
 ARG ROS_DE_VERSION
 
-# Add realm repo
+# Install all normal dependencies
 RUN apt-get update -qq \
-    && apt-get install -y curl npm \
-    && curl -s https://packagecloud.io/install/repositories/realm/realm-beta/script.deb.sh | bash
-    #&& curl -s https://packagecloud.io/install/repositories/realm/realm/script.deb.sh | bash
-    #&& curl -s https://packagecloud.io/install/repositories/realm/realm-testing/script.deb.sh | bash
+    && apt-get install -y curl
+
+# Install node using nodesource to be able to install desired major version
+# Not doing this will cause apt-get to install from 4.*
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash - \
+    && apt-get install -y nodejs build-essential
```

Comment:
What do we need build-essential for? We donøt need to build node or ROS in here, do we?

Reply:
There's a tiny bit of code that seems to need it, unfortunately. URSA, which generates keys, runs gyp and fails if there's no c++ compiler available. I do the same on the packet machines.

Reply:
I just tried it with the Node image, and URSA does require a C++ compiler, but it's there, so I changed the Dockerfile.